docstring missing for `Sampler.close()`

<!-- ⚠️ If you do not respect this template, your issue will be closed -->
<!-- ⚠️ Make sure to browse the opened and closed issues -->

### Informations

- **Qiskit Aer version**:
- **Python version**:
- **Operating system**:

### What is the current behavior?

Since https://github.com/Qiskit/qiskit-terra/commit/75673c7867222a9d8048d5264053162c5cfbe802 of Terra, `close()` is removed from `Sampler` but `AerSampler` has `close()` with assuming that docstring exists in its base class. This mismatch causes Lint error: `qiskit_aer/primitives/sampler.py:161:4: C0116: Missing function or method docstring (missing-function-docstring)`.

### Steps to reproduce the problem

Just run lint.

### What is the expected behavior?

Pass Lint.

### Suggested solutions

Add docstring to `close()` or  change `Sampler` implementation in Aer.

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -50,7 +50,7 @@ jobs:
     needs: [docs]
     strategy:
       matrix:
-        python-version: [3.7]
+        python-version: [3.8]
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}

--- a/qiskit_aer/primitives/sampler.py
+++ b/qiskit_aer/primitives/sampler.py
@@ -66,7 +66,6 @@ class Sampler(BaseSampler):
             skip_transpilation: if True, transpilation is skipped.
         """
         super().__init__(options=run_options)
-        self._is_closed = False
         self._backend = AerSimulator()
         backend_options = {} if backend_options is None else backend_options
         self._backend.set_options(**backend_options)
@@ -82,9 +81,6 @@ class Sampler(BaseSampler):
         parameter_values: Sequence[Sequence[float]],
         **run_options,
     ) -> SamplerResult:
-        if self._is_closed:
-            raise QiskitError("The primitive has been closed.")
-
         seed = run_options.pop("seed", None)
         if seed is not None:
             run_options.setdefault("seed_simulator", seed)
@@ -157,9 +153,6 @@ class Sampler(BaseSampler):
         job = PrimitiveJob(self._call, circuit_indices, parameter_values, **run_options)
         job.submit()
         return job
-
-    def close(self):
-        self._is_closed = True
 
     @staticmethod
     def _preprocess_circuit(circuit: QuantumCircuit):

--- a/test/terra/primitives/test_estimator.py
+++ b/test/terra/primitives/test_estimator.py
@@ -53,15 +53,16 @@ class TestEstimator(QiskitAerTestCase):
     def test_estimator(self, abelian_grouping):
         """test for a simple use case"""
         lst = [("XX", 1), ("YY", 2), ("ZZ", 3)]
-        with self.subTest("PauliSumOp"):
-            observable = PauliSumOp.from_list(lst)
-            ansatz = RealAmplitudes(num_qubits=2, reps=2)
-            est = Estimator(abelian_grouping=abelian_grouping)
-            result = est.run(
-                ansatz, observable, parameter_values=[[0, 1, 1, 2, 3, 5]], seed=15
-            ).result()
-            self.assertIsInstance(result, EstimatorResult)
-            np.testing.assert_allclose(result.values, [1.728515625])
+        with self.assertWarns(DeprecationWarning):
+            with self.subTest("PauliSumOp"):
+                observable = PauliSumOp.from_list(lst)
+                ansatz = RealAmplitudes(num_qubits=2, reps=2)
+                est = Estimator(abelian_grouping=abelian_grouping)
+                result = est.run(
+                    ansatz, observable, parameter_values=[[0, 1, 1, 2, 3, 5]], seed=15
+                ).result()
+                self.assertIsInstance(result, EstimatorResult)
+                np.testing.assert_allclose(result.values, [1.728515625])
 
         with self.subTest("SparsePauliOp"):
             observable = SparsePauliOp.from_list(lst)

